index.html

```diff
@@ -38,8 +41,190 @@ <h1>I'm Mucha <br> Glad to see <br> you!</h1>
+        <div class="portfolio-img">
+            <img src="images/Snapshoot-Portfolio-2.png" alt="">
+        </div>
+
+        <div class="work-content">
+
+
+            <h1 class="tonic">Multi-Post<br>Stories</h1>
+
+            <ul class="prop">
+                <li class="can">CANOPY</li>
+                <li class="bac"><img src="images/Counter.png" alt=""> Back End Dev</li>
+                <li class="num"><img src="images/Counter.png" alt=""> 2015</li>
+            </ul>
+
+            <p class="tonic-def">A daily selection of privately<br> personalized reads; no accounts or<br> sign-ups
+                required.</p>
+
+            <ul class="stack">
+                <li class="html">html</li>
+                <li class="css">css</li>
+                <li class="js">javaScript</li>
+            </ul>
+
+            <div class="button">
+                <button type="button">See Project</button>
+            </div>
+        </div>
+    </div>
+
+    <!-- card 3 -->
+    <div class="work-container3">
+        <div class="portfolio-img">
+            <img src="images/Snapshoot-Portfolio-3.png" alt="">
+        </div>
+
+        <div class="work-content">
+
+
+            <h1 class="tonic">Tonic</h1>
+
+            <ul class="prop">
+                <li class="can">CANOPY</li>
+                <li class="bac"><img src="images/Counter.png" alt=""> Back End Dev</li>
+                <li class="num"><img src="images/Counter.png" alt=""> 2015</li>
+            </ul>
+
+            <p class="tonic-def">A daily selection of privately<br> personalized reads; no accounts or<br> sign-ups
+                required.</p>
+
+            <ul class="stack">
+                <li class="html">html</li>
+                <li class="css">css</li>
+                <li class="js">javaScript</li>
+            </ul>
+
+            <div class="button">
+                <button type="button">See Project</button>
+            </div>
+        </div>
+    </div>
+
+    <!-- card-4 -->
+    <div class="work-container4">
+        <div class="portfolio-img">
+            <img src="images/Snapshoot-Portfolio-4.png" alt="">
+        </div>
+
+        <div class="work-content">
+
+
+            <h1 class="tonic">Multi-Post<br>Stories</h1>
+
+            <ul class="prop">
+                <li class="can">CANOPY</li>
+                <li class="bac"><img src="images/Counter.png" alt=""> Back End Dev</li>
+                <li class="num"><img src="images/Counter.png" alt=""> 2015</li>
+            </ul>
+
+            <p class="tonic-def">A daily selection of privately<br> personalized reads; no accounts or<br> sign-ups
+                required.</p>
+
+            <ul class="stack">
+                <li class="html">html</li>
+                <li class="css">css</li>
+                <li class="js">javaScript</li>
+            </ul>
+
+            <div class="button">
+                <button type="button">See Project</button>
+            </div>
```

Comment:
- [ ] Furthermore, we are going to use this card as the case study for the rest of the cards. You should ensure that the changes suggested on this card are also applied to the rest of the cards.
          - I have noticed that `.tonic` has a `height: 85px`, this is causing it to have a large space between the heading and the canopy paragraph. Kindly ensure to reduce or remove the height size 👍 .
          - If you are keen, you can also realize that the `.back` has a large `margin-left` kindly ensure to reduce the margin to stick to the Figma design ☺️ . In addition, ensure that the `dots` are well positioned as seen in the Figma design.
          - You managed to use Semantic HTML tags on implementing the `button`, however, to stick to the Figma design, we have to ensure that the button styling: Active, Hover, and Pressed are followed as seen in the design 🥰 .

Your design 😎 
![card](https://user-images.githubusercontent.com/99394374/219580678-390563b5-445a-4442-81b8-6ba9015a33c1.png)

Figma design 😎 
![image](https://user-images.githubusercontent.com/99394374/219581214-7d299525-6a89-4811-82da-2f040ab75a94.png)

Button design 😎 
![image](https://user-images.githubusercontent.com/99394374/219581461-72f539fe-6d9b-401c-bc61-5aa8cb80368a.png)